File: src/test/java/ru/tinkoff/load/jdbc/test/JdbcDebugTest.java
package ru.tinkoff.load.jdbc.test;

import io.gatling.javaapi.core.Simulation;

import static io.gatling.javaapi.core.OpenInjectionStep.atOnceUsers;
import static ru.tinkoff.load.jdbc.test.JdbcProtocol.dataBase;
import static ru.tinkoff.load.jdbc.test.scenarios.JdbcBasicSimulation.scn;

public class JdbcDebugTest extends Simulation {
    {
        setUp(
                scn.injectOpen(atOnceUsers(1))
        ).protocols(dataBase);
    }
}

File: src/test/kotlin/ru.tinkoff.load.jdbc.test/JdbcDebugTest.kt
package ru.tinkoff.load.jdbc.test

import io.gatling.javaapi.core.CoreDsl.*
import io.gatling.javaapi.core.*
import ru.tinkoff.load.jdbc.test.scenarios.JdbcBasicSimulation.scn


class JdbcDebugTest : Simulation()  {
    init{
            setUp(
                    scn.injectOpen(atOnceUsers(1))
            ).protocols(JdbcProtocol.dataBase)
    }
}
